### ISSUE TITLE
Find in Files does not search new unsaved files

### ISSUE THREAD

**Issue by SimpsonGSD** [slynch8/10x#1658]
Tested in 10x_notepad. Find in files doesn't seem to correctly search new unsaved files (files not on disk).

Repro:
Open notepad
Paste text in new buffer
Open Edit -> find in files
Try to search for text that exists in new buffer
No results found

**Comment by slynch8** [comment]
I agree, I'll get this fixed. Workspace Search should pick it up though.

**Comment by slynch8** [comment]
Fixed (1.0.378)
